README.md

```diff
@@ -2276,6 +2313,81 @@ The `rippled_server_status` object may have any of the following fields:
 
 # UTILITIES #
 
+## Sign Transaction ##
+[[Source]<br>](https://github.com/ripple/ripple-rest/blob/develop/api/transaction/sign.js#L50 "Source")
+
+Signs a Ripple transaction. The signed transaction can subsequently be submitted to the Ripple network with the [`submit`](#submit-transaction) endpoint. The transaction `hash` must be persisted before submitting so that the transaction can be verified.
```

Comment:
Use second person active voice instead of passive voice here, e.g. "you can submit the signed transaction" instead of "the signed transaction can be submitted". Same goes for "the hash must be persisted".